test/evergreen.yml

```diff
@@ -2919,6 +2947,69 @@ tasks:
 
             # Run S3 extension unit testing.
             ext/storage_sources/s3_store/test/run_s3_unit_tests
+  - name: azure-tiered-unittest-test
+    tags: ["pull_request"]
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          <<: *configure_flags_tiered_storage_azure
+          HAVE_UNITTEST: -DHAVE_UNITTEST=1
+          cmake_binary: '/opt/mongodbtoolchain/v4/bin/cmake'
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/cmake_build"
+          shell: bash
+          script: |
+            set -o errexit
+            export AZURE_STORAGE_CONNECTION_STRING="${azure_sdk_s3_ext_access_key}"
+            set -o verbose
+
+            # Run AZURE extension unit testing.
+            ext/storage_sources/azure_store/test/run_azure_unit_tests
+  - name: azure-gcp-tiered-test-small
+    tags: ["pull_request", "python"]
+    commands:
+      - func: "get project"
+      - func: "install gcp dependencies"
+      - func: "compile wiredtiger"
+        vars:
+          <<: *configure_flags_tiered_storage_gcp
```

Comment:
Let's check if we need Azure flags here as well. We might need to have a separate task for Azure